interfaces/mount/change_test.go

```diff
@@ -174,3 +194,54 @@ func (s *changeSuite) TestNeededChangesSmartEntryComparison(c *C) {
 		{Entry: mount.Entry{Dir: "/a/b/c"}, Action: mount.Mount},
 	})
 }
+
+// Change.Perform calls the mount system call.
+func (s *changeSuite) TestPerformMount(c *C) {
+	chg := &mount.Change{Action: mount.Mount, Entry: mount.Entry{Name: "source", Dir: "target", Type: "type"}}
+	err := chg.Perform()
+	c.Assert(err, IsNil)
```

Comment:
c.Assert(chg.Perform(), IsNil)

applies to asserts further down too.

Reply:
Ack